<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWidget</class>
    <widget class="QWidget" name="MainWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWidget</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <widget class="QSplitter" name="splitter">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <widget class="QWidget" name="leftPane" native="true">
                        <layout class="QVBoxLayout" name="verticalLayoutLeft">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QListWidget" name="chatList" />
                            </item>
                            <item>
                                <widget class="QPushButton" name="newChatButton">
                                    <property name="text">
                                        <string>New</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="rightPane" native="true">
                        <layout class="QVBoxLayout" name="verticalLayoutRight">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QListWidget" name="chatContent" />
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QPushButton" name="uploadButton">
                                            <property name="text">
                                                <string>Upload</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="recordButton">
                                            <property name="text">
                                                <string>Record</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>
